### ISSUE TITLE
nteract_on_jupyter not working (404 : Not Found)

### ISSUE THREAD

**Issue by dimitry-ishenko** [nteract/archived-desktop-app#5083]
**Application or Package Used**
nteract_on_jupyter

**Describe the bug**
Trying to run `jupyter nteract` results in:

404 : Not Found
You are requesting a page that does not exist!

**To Reproduce**

1. Trying to follow instructions in the [README.md](https://github.com/nteract/nteract/blob/master/applications/jupyter-extension/README.md) miseably fails on: `pip install nteract_on_jupyter`
2. Turns out the package is called _nteract-on-jupyter_. Notice the dashes instead of underlines.
3. `pip install nteract-on-jupyter` seems to work
4. `jupyter nteract` (or `jupyter-nteract`) launches fine
5. You are greeted with a nice cup of "404 : Not Found"

**Expected behavior**
Some sort of a page where I can open a notebook and run it would be nice. Ideally much less ugly than Jupyter's Notebook and Lab interfaces (which is the whole point of me trying it out).

**Screenshots**
![Screenshot from 2020-05-05 12-43-01](https://user-images.githubusercontent.com/2858291/81092378-8096e000-8ece-11ea-8cdb-df838051b6ed.png)

**Desktop (please complete the following information):**

- OS: Ubuntu 20.04 LTS
- Browser: Google Chrome
- Version: 81.0.4044.129 (Official Build) (64-bit)

**Comment by captainsafia** [comment]
Have you enabled the nteract_on_jupyter server extension?

```
jupyter serverextension enable nteract_on_jupyter
```

As a sidenote: the nteract_on_jupyter application is nearing deprecation. We're currently focusing our efforts on the desktop app and the new nteract web app, which will have the same features as the nteract_on_jupyter extension (connection to Jupyter kernels via a Jupyter server API) and some more goodies (connecting to Jupyter sessions on Binder).

I hope this was helpful.

**Comment by dimitry-ishenko** [comment]
Thanks @captainsafia. That's definitely helpful!

> Have you enabled the nteract_on_jupyter server extension?

I sure haven't, since it was mentioned in the Development section, which I didn't bother to read. :stuck_out_tongue_closed_eyes: OK, so I got it working, but was a bit underwhelmed. I guess I've expected something that looks similar to the atom/hydrogen combination.

I've tried the desktop app briefly, but it seems somewhat limited. There is certain functionality I am after (which I've described in another issue), so I guess hydrogen is my best bet for the moment.

Thanks again.
D-mo
